app/views/transactions/show.html.erb

```diff
@@ -38,5 +38,5 @@
   <%= @transaction.actualized_at %>
 </p>
 
-<%= link_to 'Edit', edit_transaction_path(@transaction) %> |
```

Comment:
We should hide this only for actualized transactions.  Else, we want the link to show, right?  Or am I missing something?